changelog_entry.yaml

```
@@ -0,0 +1,4 @@
+- bump: minor
```

Since no new features added
```suggestion
- bump: patch
```